src/azure-cli/azure/cli/command_modules/mysql/tests/latest/test_mysql_scenario.py

```diff
@@ -2169,3 +2171,77 @@ def _test_identity_aad_admin_mgmt(self, database_engine, resource_group, passwor
         # delete everything
         for server_name in [replica[0], replica[1], server]:
             self.cmd('{} flexible-server delete -g {} -n {} --yes'.format(database_engine, resource_group, server_name))
+
+class MySQLExportTest(ScenarioTest):
+    profile = None
+
+    def get_current_profile(self):
+        if not self.profile:
+            self.profile = self.cmd('cloud show --query profile -otsv').output
+        return self.profile
+
+    def get_account_key(self, group, name):
+        if self.get_current_profile() == '2017-03-09-profile':
+            template = 'storage account keys list -n {} -g {} --query "key1" -otsv'
+        else:
+            template = 'storage account keys list -n {} -g {} --query "[0].value" -otsv'
+
+        return self.cmd(template.format(name, group)).output
+
+    def get_account_info(self, group, name):
+        """Returns the storage account name and key in a tuple"""
+        return name, self.get_account_key(group, name)
```

Comment:
```suggestion
```
The default profile is always `latest`. And you don't need to get storage account info by yourself, `StorageAccountPreparer()` already support returning account info